src/db.rs

```diff
@@ -186,6 +200,16 @@ impl Database {
             }
         }
 
+        // Populate full-text search table
+        info!("Populating FTS table...");
+        db.execute(
+            r#"
+            INSERT INTO messages_fts (content, username, avatar, messages_rk)
```

Comment:
Not familiar with _rk naming, is there a reason we can't still just call it rowid? Virtual tables do not include one, but does it have to be named something else? Or maybe messages_rowid to be more verbose?

Reply:
For some reason I was convinced it's called `rowkey` instead of `rowid` when I went with `_rk`, but FTS tables do have [their own rowid](https://sqlite.org/fts5.html) so we can't reuse that. Happy to change it to something else.

Reply:
ok, messages_rowid should be fine then I think